.github/workflows/cd.yml

```diff
@@ -11,13 +11,21 @@ jobs:
 
     steps:
 
-    - name: Set up JDK 11
+    - name: Set up JDK 14
       uses: actions/setup-java@v1
       with:
-        java-version: 11.0.x
-        server-id: sonatype-nexus-staging
-        server-username: SONATYPE_USER
-        server-password: SONATYPE_PASSWORD
+        java-version: 14.0.x
+
+    - name: Checkout OpenSearch
+      uses: actions/checkout@v2
+      with:
+        repository: 'opensearch-project/OpenSearch'
+        path: OpenSearch
+        ref: '1.0'
+
+    - name: Build OpenSearch
```

Comment:
Should this step be after `Cache Maven packages` step?

Reply:
Shouldn't OpenSearch artifact built be cached? 
If moving `Build OpenSearch` step and putting it after `Cache Maven packages` step, both of `Build OpenSearch` and `Build` steps will be after `Cache Maven packages` step. Shouldn't the build outputs be cached?

Reply:
What is the point of caching OpenSearch maven packages when they are rebuild anyway?

Reply:
Agree. In this case, only downloaded dependencies should be cached. There is no point of caching build outputs. 
Moved. Thanks!